<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>799</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MPT Chart</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>mpt.png</normaloff>mpt.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="chartTab">
       <attribute name="title">
        <string>Chart</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QChartView" name="chartView"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="dataTab">
       <attribute name="title">
        <string>Data</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QPlainTextEdit" name="dataLog"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QPlainTextEdit" name="statusLog">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>100</height>
       </size>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>799</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="fileMenu">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionExportCSV"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="helpMenu">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAboutQt"/>
   </widget>
   <widget class="QMenu" name="deviceMenu">
    <property name="title">
     <string>Device</string>
    </property>
    <addaction name="actionConnect"/>
    <addaction name="actionDisconnect"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionPulse"/>
   </widget>
   <addaction name="fileMenu"/>
   <addaction name="deviceMenu"/>
   <addaction name="menuView"/>
   <addaction name="helpMenu"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionAboutQt">
   <property name="text">
    <string>About Qt</string>
   </property>
  </action>
  <action name="actionSerialPort">
   <property name="text">
    <string>Port</string>
   </property>
  </action>
  <action name="actionConnect">
   <property name="text">
    <string>Connect</string>
   </property>
   <property name="shortcut">
    <string>F5</string>
   </property>
  </action>
  <action name="action300">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>300</string>
   </property>
  </action>
  <action name="action1200">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>1200</string>
   </property>
  </action>
  <action name="action4800">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>4800</string>
   </property>
  </action>
  <action name="action9600">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>9600</string>
   </property>
  </action>
  <action name="action19200">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>19200</string>
   </property>
  </action>
  <action name="action38400">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>38400</string>
   </property>
  </action>
  <action name="action57600">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>57600</string>
   </property>
  </action>
  <action name="action74880">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>74880</string>
   </property>
  </action>
  <action name="action115200">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>115200</string>
   </property>
  </action>
  <action name="action230400">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>230400</string>
   </property>
  </action>
  <action name="action250000">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>250000</string>
   </property>
  </action>
  <action name="action500000">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>500000</string>
   </property>
  </action>
  <action name="action1000000">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>1000000</string>
   </property>
  </action>
  <action name="action2000000">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>2000000</string>
   </property>
  </action>
  <action name="actionDisconnect">
   <property name="text">
    <string>Disconnect</string>
   </property>
   <property name="shortcut">
    <string>F6</string>
   </property>
  </action>
  <action name="actionPulse">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Pulse</string>
   </property>
   <property name="shortcut">
    <string>F9</string>
   </property>
  </action>
  <action name="actionExportCSV">
   <property name="text">
    <string>Export CSV</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QChartView</class>
   <extends>QGraphicsView</extends>
   <header>QtCharts</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
